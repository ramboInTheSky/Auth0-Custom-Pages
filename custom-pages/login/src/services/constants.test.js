import Enzyme from 'enzyme'
import Adapter from 'enzyme-adapter-react-14'

import authOpts from './constants'

Enzyme.configure({ adapter: new Adapter() })

describe('Register components', () => {
  it('should retrieve the constants correctly', () => {
    expect(authOpts).toMatchSnapshot()
  })
})
