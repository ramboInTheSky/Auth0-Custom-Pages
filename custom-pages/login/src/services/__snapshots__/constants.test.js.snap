// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Register components should retrieve the constants correctly 1`] = `undefined`;
